src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -556,7 +531,7 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 TimeSpan timerInterval = timer == null ? TimeSpan.MinValue : TimeSpan.FromMilliseconds(timer.Interval);
                 try
                 {
-                    await InvokeAsync(invoker, invokeParameters, timeoutTokenSource, functionCancellationTokenSource,
+                    await InvokeAsync(invoker, parameterHelper, timeoutTokenSource, functionCancellationTokenSource,
```

Comment:
For Hamza's PR which drove your changes, the helper will provide access to both parameter names and values now?